### PR TITLE
[stack-switching] OpCode addition

### PR DESCRIPTION
**Pull request by davidmaamoaix**
This PR includes the changes to `OpCode.v3` following previous meetings on implementing the stack-switching proposal.

### CODE REVIEW

**Comment by davidmaamoaix** [comment]
Hi! I just finished the placeholder visitors. Please let me know if there's further edits that I need to do.

File: src/engine/BytecodeIterator.v3 [hunk 1]
```diff
@@ -81,12 +81,17 @@ class BytecodeIterator {
...
 				I32,
-				I64 => codeptr.skip_leb();
+				I64,
+				CONT_INDEX => codeptr.skip_leb();
 				VALUE_TYPES => codeptr.skip_value_types();
 				LABELS => {
 					var count = codeptr.read_uleb32();
 					for (i < count + 1) codeptr.skip_leb();
 				}
+				HANDLERS => {
```
Comment (titzer) [inline src/engine/BytecodeIterator.v3:91]: Are the handlers a list of pairs (tag_index, label), i.e. two LEBs?

**Review by titzer — APPROVED** [review]
Looks good with just one minor comment.

File: src/engine/BytecodeIterator.v3 (hunk 1 above)
Comment (davidmaamoaix) [reply src/engine/BytecodeIterator.v3:91]: My bad. This should be fixed in the new commit.

**Review by titzer — APPROVED** [review]

**Pull request merged**